src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar__month-length--28">
+        <div class="calendar-day calendar-day__start-day--fri"></div>
```

Comment:
all the modifiers should be related to the calendar itself

```suggestion
      <div class="calendar calendar--month-length-28 calendar--start-day-fri">
        <div class="calendar-day"></div>
```